test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_Node_MultipleProcesses.cs

```diff
@@ -19,20 +19,19 @@ namespace Microsoft.Azure.WebJobs.Script.Tests.EndToEnd
 {
     [Trait(TestTraits.Category, TestTraits.EndToEnd)]
     [Trait(TestTraits.Group, TestTraits.SamplesEndToEnd)]
-    public class SamplesEndToEndTests_Node_MultipleProcesses : IAsyncDisposable
+    public class SamplesEndToEndTests_Node_MultipleProcesses : IAsyncLifetime
     {
-        private readonly MultiplepleProcessesTestFixture _fixture;
+        private readonly MultipleProcessesTestFixture _fixture;
```

Comment:
Can inline this and remove the ctor
```suggestion
        private readonly MultipleProcessesTestFixture _fixture = new();
```